catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchFilterVisitor.java

```diff
@@ -271,6 +270,23 @@ public Object visit(Intersects filter, Object data) {
         return super.visit(filter, data);
     }
 
+    private String buildPolygonWkt(Coordinate[] coords) {
```

Comment:
We could use the WKT writer here instead of generating the WKT String ourselves : https://locationtech.github.io/jts/javadoc/org/locationtech/jts/io/WKTWriter.html